src/applets/status/BluetoothIndicator.vala

```diff
@@ -21,7 +21,10 @@ public class BluetoothIndicator : Gtk.Bin {
 	public Gtk.Image? image = null;
 
 	public Gtk.EventBox? ebox = null;
+#if USE_OPENBSD
```

Comment:
With regards to the Bluetooth stuff, I think it would be preferable to add a more general Meson option: `with-bluetooth`, defaulting to `true`. Then any operating system that doesn't ship gnome-bluetooth can set that option.